Dockerfile.ci

```diff
@@ -3,16 +3,18 @@ FROM elifesciences/proofreader-php:latest AS proofreader
 FROM elifesciences/annotations_cli:${commit}
 USER root
 
-# extract into another image
-RUN git clone https://github.com/asm89/smoke.sh /opt/smoke.sh
-
 RUN mkdir -p build && chown www-data:www-data build
 
 USER elife
 COPY --from=proofreader --chown=elife:elife /srv/proofreader-php /srv/proofreader-php
 RUN ln -s /srv/proofreader-php/bin/proofreader /srv/bin/proofreader
 
-USER www-data
-COPY .php_cs project_tests.sh smoke_tests.sh /srv/annotations/
+USER root
+RUN composer --no-interaction install --no-suggest --no-autoloader
 
-CMD ["/bin/bash", "project_tests.sh"]
+COPY tests/ /srv/annotations/tests/
+RUN composer dump-autoload --classmap-authoritative
```

Comment:
Should probably the last provisioning step?

Reply:
The sooner it is done, the more it can be cached; but in the case of `ci` it doesn't matter a great deal, as it is almost always being entirely re-executed (except for replayed builds)

Reply:
Thought it to be _less_ cacheable than the below files, but I guess if the `src` or `vendor` has changed anyway nothing's cached anymore so doesn't matter...

Reply:
Going for clarity, I'll put the `COPY` together, pushing the autoload down as the last step that is executed as `root` (later on will be executed as `elife`).